src/services/ShSimulatorService.tsx

```diff
@@ -87,6 +91,18 @@ export interface RestSimulationContext {
 	) => Promise<FullSimulationData | undefined>;
 }
 
+const recreateOrderInEstimators = (
```

Comment:
wasn't that removed in the previous PR ? new backend should handle the order I believe